schema-mapping/data-types/core/ExternalReference.json

```diff
@@ -0,0 +1,27 @@
+{
+  "identifier": "core:ExternalReference",
+  "table": "property",
+  "properties": [
+    {
+      "name": "targetResource",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "type": "core:URI"
+    },
+    {
+      "name": "informationSystem",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "value": {
+        "column": "val_codespace",
+        "type": "uri"
+      }
+    },
+    {
+      "name": "relationType",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "value": {
+        "column": "val_string",
+        "type": "uri"
```

Comment:
the type for `relationType` here should be `string`?

Reply:
According to CityGML 3, `relationType` is of type "URI". We just use `val_string` to store the URI because `val_uri` is already used for `targetResource`.